lib/Doctrine/DBAL/Driver/SQLSrv/SQLSrvStatement.php

```diff
@@ -150,9 +150,13 @@ public function bindParam($column, &$variable, $type = null, $length = null)
      */
     public function closeCursor()
     {
-        if ($this->stmt) {
```

Comment:
@morozov basically it is about that yes. But the test case I used was this:

```
1) Doctrine\Tests\DBAL\Functional\StatementTest::testReuseStatementAfterClosingCursor
Failed asserting that 1 matches expected 2.

/php/srv/tests/Doctrine/Tests/DBAL/Functional/StatementTest.php:151
```

But also we are talking about SQL Server here, not DB2.